### ISSUE TITLE
[v1.0-Elements-Column.Size.Ratio.json]: Text 'Column 1 (10)'  is not visible properly in 'Column.Size.Ratio.json' card.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7427]
https://microsoft.visualstudio.com/OS/_workitems/edit/37811613
